draft-ietf-cdni-uri-signing.xml

```diff
@@ -544,6 +544,33 @@
             claim if no Client IP claim existed in the received
             signed JWT.</t>
         </section>
+        <section anchor="cdnib_claim" title="CDNI Base (cdnib) claim">
+            <t>Base (cdnir) [optional] -
+            The base path for which the token is intended. Its type is a
+            JSON string. It contains a <xref target="RFC3986">URI</xref>,
+            except that any or all of the scheme, host, or port MAY be
+            omitted. This URI MUST NOT contain User Information
+            (<xref target="RFC3986" /> Section 3.2.1), Query (Section 3.4),
+            or Fragment (Section 3.5) components.</t>
+
+            <t>The scheme is terminated by a colon (':'), which MUST be
+            omitted if the scheme is. The authority is preceeded by a
+            double slash ("//"), which MUST be omitted if the authority
+            is, and terminated by the next slash ("/"), which begins the
```

Comment:
I found this confusing. I assume the intent is "... MUST be omitted if the authority is not present".

Reply:
Yeah, this can be cleaned up. Originally, it just indicated that the various sections were optional, but then parsing is ambiguous. By specifying which delimiters must be omitted along with the sections they delimit, you ensure that there's no ambiguity.